ledger-tool/src/bigtable.rs

```diff
@@ -329,6 +390,42 @@ impl BigTableSubCommand for App<'_, '_> {
                                 .help("Maximum number of slots to return"),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("compare-blocks")
+                        .about("Compare a list of slots with confirmed blocks for the given range with a standard source. \
+                                Output missing blocks.")
+                        .arg(
+                            Arg::with_name("starting_slot")
+                                .long("starting-slot")
+                                .validator(is_slot)
+                                .value_name("SLOT")
+                                .takes_value(true)
+                                .index(1)
+                                .required(true)
+                                .default_value("0")
+                                .help("Start listing at this slot"),
+                        )
+                        .arg(
+                            Arg::with_name("limit")
+                                .long("limit")
+                                .validator(is_slot)
+                                .value_name("LIMIT")
+                                .takes_value(true)
+                                .index(2)
+                                .required(true)
+                                .default_value("1000")
+                                .help("Maximum number of slots to return"),
+                        ).arg(
+                            Arg::with_name("credential")
+                            .long("credential")
+                            .value_name("CREDENTIAL")
+                            .takes_value(true)
+                            .index(3)
+                            .required(true)
+                            .default_value("")
+                            .help("File path of credential with a standard source"),
+                        ),
```

Comment:
```suggestion
                        )
                        .arg(
                            Arg::with_name("credential")
                                .long("credential")
                                .value_name("CREDENTIAL")
                                .takes_value(true)
                                .index(3)
                                .required(true)
                                .default_value("")
                                .help("File path of credential with a standard source"),
                        ),
```
cargo fmt sometimes pukes on long clap blocks